benchmarks/asv.conf.venv.json

```diff
@@ -0,0 +1,95 @@
+        // (for git) or "tip" (for mercurial).
+        // "branches": ["master"], // for git
+        // "branches": ["tip"],    // for mercurial
+
+        // The DVCS being used.  If not set, it will be automatically
+        // determined from "repo" by looking at the protocol in the URL
+        // (if remote), or by looking for special directories, such as
+        // ".git" (if local).
+        // "dvcs": "git",
+
+        // The tool to use to create environments.  May be "conda",
+        // "virtualenv" or other value depending on the plugins in use.
+        // If missing or the empty string, the tool will be automatically
+        // determined by looking for tools on the PATH environment
+        // variable.
+        "environment_type": "virtualenv",
+
+        // the base URL to show a commit for the project.
+        "show_commit_url": "https://github.com/colour-science/colour/commits/develop",
+
+        // The Pythons you'd like to test against.  If not provided, defaults
+        // to the current version of Python used to run `asv`.
+        //"pythons": ["2.7", "3.6"],
+
+        // The matrix of dependencies to test.  Each key is the name of a
+        // package (in PyPI) and the values are version numbers.  An empty
+        // list indicates to just test against the default (latest)
+        // version.
+        "matrix": {
+        "pip+biblib-simple":[],
+        "coverage":[],
+        "coveralls":[],
+        "flake8":[],
+        "invoke":[],
+        "jupyter":[],
+        "mock":[],
+        "pip+nbformat":["4.1"],
+        "nose":[],
+        "pip+pre-commit":[],
+        "pytest":[],
+        "pip+restructuredtext-lint":[],
+        "sphinx":[],
+        "sphinx_rtd_theme":[],
+        "pip+sphinxcontrib-bibtex":[],
+        "toml":[],
+        "twine":[],
+        "pip+yapf":["0.23"],
+        "imageio":[],
+        "scipy":[],
+        "six":[],
+        "poetry":[]
+    },
+
+
+    // The directory (relative to the current directory) that benchmarks are
+    // stored in.  If not provided, defaults to "benchmarks"
+    // "benchmark_dir": "benchmarks",
+
+    // The directory (relative to the current directory) to cache the Python
+    // environments in.  If not provided, defaults to "env"
+    // "env_dir": "env",
+
+
+    // The directory (relative to the current directory) that raw benchmark
+    // results are stored in.  If not provided, defaults to "results".
+    // "results_dir": "results",
+
+    // The directory (relative to the current directory) that the html tree
+    // should be written to.  If not provided, defaults to "html".
+    // "html_dir": "html",
+
+    // The number of characters to retain in the commit hashes.
+    // "hash_length": 8,
+
+    // `asv` will cache wheels of the recent builds in each
+    // environment, making them faster to install next time.  This is
+    // number of builds to keep, per environment.
+    // "wheel_cache_size": 0
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/manual/colour-science/colour/pullRequest?prid=5223583)